rabbitmq.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: all
+  pre_tasks:
+    - name: Install pip
+      easy_install: name=pip state=latest
+      sudo: yes
+    - name: Install boto
+      pip: name=boto
+      sudo: yes
+  vars:
+    rabbitmq_ec2_tag_value: rabbitmq-dev
+    rabbitmq_ec2_region: eu-central-1
+    rabbitmq_cookie: HWIKAFRWMIKYUNPMJHGW
+    rabbitmq_cluster: true
+    rabbitmq_hostname: "{{ ansible_hostname }}"
+    rabbitmq_nodename_suffix:
+    app_settings:
+      rabbitmq:
+        aws_access_key_id: AKIAJWSARQGWPYLPGY3Q
+        aws_secret_access_key: S4EF+aY/w7UrYPb1gMolj8rpOloVlBU65tbSW6iv
```

Comment:
I think you added your keys on accident may want to remove them if aws hasn't already.